docs/en/stack/ml/df-analytics/evaluatedf-api.asciidoc

```diff
@@ -0,0 +1,100 @@
+[role="xpack"]
+[[ml-dfanalytics-evaluate]]
+== Evaluating {dfanalytics}
+
+Using the {dfanalytics} features to gain insights from a data set is an 
+iterative process. You might need to experiment with different analyses, 
+parameters, ways to transform data before you arrive to a result that satisfies 
+your use case. A valuable companion to this process is the 
+{ref}/evaluate-dfanalytics.html[{evaluatedf-api}] that allows you to evaluate 
+the {dfanalytics} performance against a marked up data set. It helps to 
+understand error distributions and to know the points where the {dfanalytics} 
+model performing well or less trustworthily.
+
+{evaluatedf-api-cap} is designed for providing a general evaluation mechanism 
+for the different kinds of {dfanalytics}. That means that every type of analysis 
```

Comment:
> That means that every type of analysis...

This sentence seems strange to me, since the API only seems to have one type of evaluation at this time. Perhaps it should be omitted until there are other types?